charts/aws-efs-csi-driver/values.yaml

```diff
@@ -112,6 +113,11 @@ node:
     #   cpu: 100m
     #   memory: 128Mi
   nodeSelector: {}
+  updateStrategy:
+    {}
```

Comment:
Aside from this `{}` which might be put on the top line the only thing this PR is missing, IMHO, is the fact that it is missing a bump of the chart version.

Reply:
good comments @pierluigilenoci , applied to new commit. 
thanks